### PR TITLE
CNV-4091 - RN feature - nmstate

### PR DESCRIPTION
**Pull request by aburdenthehand**
Added release note for cnv-4091 - nmstate feature

### CODE REVIEW

**Review by phoracek — APPROVED** [review]
/lgtm

**Comment by yossisegev** [comment]
LGTM (QE)

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 1]
```diff
@@ -17,7 +17,9 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 // CNV-2928/CNV-4166 OVN-Kubernetes network provider
 
-// CNV-4091 nmstate
+* {CNVProductNameStart} uses link:https://nmstate.github.io/[`nmstate`] to report on and configure the state of the node network. This makes it possible to modify network policy configuration, to create Linux bridges, bonds, and VLAN devices on all nodes, by applying a single configuration manifest to the cluster.
```
Comment (ousleyp) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:20]: The second sentence is a bit too hard to read. I'd try splitting it into two sentences, and I'd also:
- s/This makes it possible to modify/You can change

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 2]
```diff
@@ -17,7 +17,9 @@ include::modules/technology-preview.adoc[leveloffset=+2]
...
+
+For more information, see the xref:../../cnv/cnv_node_network/cnv-observing-node-network-state.adoc#cnv-about-nmstate_cnv-observing-node-network-state[node networking] chapter.
```
Comment (ousleyp) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:22]: I think this needs to be a sub-bullet.

**Review by ousleyp — COMMENTED** [review]
Just a couple things to look at before merge. :)

**Pull request merged**